subtests/docker_cli/liverestore/liverestore.py

```diff
@@ -0,0 +1,185 @@
+r"""
+Summary
+---------
+
+Test live-restore: running container should survive docker daemon restart.
+
+Operational Summary
+----------------------
+
+#. Run a docker container
+#. Restart docker daemon
+#. Verify that container is listed under 'docker ps' and is still running.
```

Comment:
It checks that the container is now a child of init, not of dockerd. I suppose I could preserve the container PID and make sure it doesn't change. I'll add that for next revision.